packages/core/src/sync/events.ts

```diff
@@ -308,7 +316,9 @@ export const buildEvents = ({
                   trace: convertTrace(trace),
                   block: convertBlock(block),
                   transaction: convertTransaction(transaction),
-                  transactionReceipt: undefined,
+                  transactionReceipt: shouldGetTransactionReceipt(filter)
```

Comment:
IIRC we talked about adding the receipt properties to the `transaction` object - what's your latest thinking?

Reply:
Gonna do it in a separate pr